test/validator.test.ts

```diff
@@ -114,142 +92,94 @@ describe('Validator', () => {
           content: 'Another parameter description',
         },
       ],
-      returns: [],
-    };
-
+    });
     const result = validator.validate(node, natspec);
     expect(result).toContainEqual(`@return ${paramName} is missing`);
   });
 
   it('should reveal missing natspec for unnamed returned values', () => {
     node = contract.vFunctions.find(({ name }) => name === 'externalSimpleMultipleReturn')!;
-    let natspec = {
+    const natspec = mockNatspec({
       tags: [
         {
           name: 'notice',
           content: 'External function that returns a bool',
         },
-        {
-          name: 'dev',
-          content: 'A dev comment',
-        },
-      ],
-      params: [
-        {
-          name: '_magicNumber',
-          content: 'A parameter description',
-        },
-        {
-          name: '_name',
-          content: 'Another parameter description',
-        },
       ],
       returns: [
         {
           name: '_isMagic',
           content: 'Some return data',
         },
       ],
-    };
+    });
 
     const result = validator.validate(node, natspec);
     expect(result).toContainEqual(`@return missing for unnamed return №2`);
   });
 
   it('should warn of a missing unnamed return', () => {
     node = contract.vFunctions.find(({ name }) => name === 'externalSimpleMultipleUnnamedReturn')!;
-    let natspec = {
+    const natspec = mockNatspec({
       tags: [
         {
           name: 'notice',
           content: 'External function that returns a bool',
         },
-        {
-          name: 'dev',
-          content: 'A dev comment',
-        },
       ],
-      params: [],
       returns: [
         {
           name: 'Some',
           content: 'return data',
         },
       ],
-    };
+    });
 
     const result = validator.validate(node, natspec);
-    expect(result).toEqual([`@return missing for unnamed return №2`]); // only 1 warning
+    expect(result).toContainEqual(`@return missing for unnamed return №2`);
```

Comment:
when we change straight equals to contains, it might pass this case for example: 
> [`@return missing for unnamed return №1`, `@return missing for unnamed return №2`]
but we want to check if it's straight 1 warning and no more